vmngclient/api/templates.py

```diff
@@ -125,8 +127,13 @@ def attach(self, name: str, device: Device) -> bool:
         """
         try:
             template_id = self.get_id(name)
+            self.__validation_template(template_id, device=device)
         except NotFoundError:
             return False
```

Comment:
How about adding here:
```python
logger.error(f"Error, Template with name {name} not found on {device}")
```
?

Or something similar, just to log for the users why it was not attached.